docs/architecture.md

```diff
@@ -1,18 +1,18 @@
-# Architecture of CRI-Containerd
-This document describes the architecture of `cri-containerd`.
+# Architecture of The CRI Plugin
+This document describes the architecture of the `cri` plugin for `containerd`.
 
-Cri-containerd is a containerd based implementation of Kubernetes [container runtime interface (CRI)](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/cri/runtime/v1alpha2/api.proto). It operates on the same node as the [Kubelet](https://kubernetes.io/docs/reference/generated/kubelet/) and [containerd](https://github.com/containerd/containerd). Layered between Kubernetes and containerd, cri-containerd handles all CRI service requests from the Kubelet and uses containerd to manage containers and container images.
+This plugin is an implementation of Kubernetes [container runtime interface (CRI)](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/cri/runtime/v1alpha2/api.proto). Containerd operates on the same node as the [Kubelet](https://kubernetes.io/docs/reference/generated/kubelet/). The `cri` plugin inside containerd handles all CRI service requests from the Kubelet and uses containerd internals to manage containers and container images.
```

Comment:
As a part of containerd, `cri` plugin operates on the same node as the [Kubelet](https://kubernetes.io/docs/reference/generated/kubelet/). It handles all CRI service requests from the Kubelet and uses containerd internals to manage containers and container images.

Reply:
I like it will use that on the rewrite unless you want to do the readme rewrite.